src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,50 @@
+package lotto.domain;
+
+import lotto.exception.IllegalLottoNumberCountException;
+import lotto.util.LottoNumber;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class LottoNumbers {
+
+    private final List<Integer> lottoNumbers;
```

Comment:
```suggestion
    private final Set<LottoNumber> lottoNumbers;
```
- 이렇게 LottoNumber라는 원시값 포장 객체를 만들어서 사용해보면 어떨까요?
- List 대신에 Set을 사용해보면 어떨까요?
- LottoNumber의 순서 보장을 위해서 TreeSet을 이용해보세요!
- Set = 인터페이스, TreeSet = 구현체

Reply:
적용해보겠습니다!